config.json

```diff
@@ -1,42 +1,54 @@
 {
-    "web": {
-        "port": 3000,
-        "title": "BitCannon",
-        "torrentsPerPage": 100,
-        "defaultSearchSorting": "seeders",
-        "defaultSearchOrder": "desc"
-    },
-    "database": {
-        "mongodb": {
-            "enabled": true,
-            "host": "localhost",
-            "port": 27017,
-            "collection": "bitcannon"
-        }
+  "web": {
+    "port": 3000,
+    "title": "BitCannon",
+    "torrentsPerPage": 100,
+    "defaultSearchSorting": "seeders",
+    "defaultSearchOrder": "desc"
+  },
+  "database": {
+    "mongodb": {
+      "enabled": true,
+      "host": "localhost",
+      "port": 27017,
+      "collection": "bitcannon"
+    }
+  },
+  "torrents": {
+    "whitelist": {
+      "enabled": false,
+      "categories": []
     },
-    "session": {
-        "secret": "asdasdasdasd43jn5kj2n43k"
+    "blacklist": {
+      "enabled": false,
+      "categories": []
+    }
+  },
+  "session": {
+    "secret": ""
```

Comment:
I'd like to move to JWT if possible so we can move the UI to using Vue as that's what I'm using on all other projects. It'd allow the server as well as the background processes to run on a seedbox, etc. and have the ui bundled as an Electron app or even just hosted locally via Apache, Nginx, Caddy, etc.